### DISCUSSION TITLE
Renaming and/or removing codelists

### DISCUSSION THREAD

**Discussion by rmgpanw** [opensafely/documentation#438] [Codelists] [answered]
Hi, thanks for creating this amazing resource. Is there a way to:

1. Rename an existing codelist
2. Delete a codelist

Thank you!

**Comment by inglesp** [comment, marked as answer]
This isn't exposed in the UI (but should be).  If there are codelists you'd like to rename or delete, let me know here and I'll make the changes.

**Reply by rmgpanw** [reply]
Thanks @inglesp, I don't have any codelists that need urgent editing/removing so won't bother you with this for now. Will get back to you if this changes in future
